build.gradle

```diff
@@ -1921,6 +1921,7 @@ project(':streams') {
     testImplementation project(':clients').sourceSets.test.output
     testImplementation project(':core')
     testImplementation project(':core').sourceSets.test.output
+    testImplementation project(':server-common').sourceSets.test.output
```

Comment:
The streams integration harness has a transitive runtime dependency to `Scheduler/KafkaScheduler` and dependencies to test jars are not transitive.